### DISCUSSION TITLE
Mongoose support for QNX

### DISCUSSION THREAD

**Discussion by FaiqueAli** [cesanta/mongoose#2818] [Ask questions, get answers] [answered]
Hi,
I am looking for C++ api that I can use to create a REST API.
Before start working with mongoose, I would like to know that does mongoose supports has support for qcc compiler, as we have to create the API for QNX OS. we already have a project build and running on QNX, at this stage we need to create rest api that can be build using QNX qcc compiler and then I can use it in QNX OS.

**Comment by scaprile** [comment, marked as answer]
There are no examples for that, hence we've never tried that, then we can't tell you if it works or not with that compiler. It will depend on its peculiarities. Of course there is no built-in specific support for QNX, it may work if it behaves like yet another un*x-like.
